app/scripts/modules/core/delivery/executions/executions.less

```diff
@@ -85,4 +85,8 @@ executions {
     fill: @stage-running;
     background-color: @stage-running;
   }
+  &.execution-marker-skipped {
+    fill: @stage-succeeded;
```

Comment:
Wasn't convinced we needed a different color to indicate skipped stages, thoughts?